### PR TITLE
Update account interface

### PR DESCRIPTION
**Pull request by ericnordelo**
Replaces #638

### CODE REVIEW

File: src/openzeppelin/account/interface.cairo [hunk 1]
```diff
@@ -12,10 +13,12 @@ struct Call {
     calldata: Array<felt252>
 }
 
-trait IAccount {
+trait IStandardAccount {
```
Comment (andrew-fleming) [inline src/openzeppelin/account/interface.cairo:16]: Hmm should the trait be `ISRC6` given the forthcoming SNIP?

File: src/openzeppelin/utils/constants.cairo [hunk 2]
```diff
@@ -6,8 +6,9 @@
 // See: https://github.com/starknet-io/SNIPs/pull/24
 const ISRC5_ID: felt252 = 0x3f918d17e5ee77373b56385708f855659a07f75997f365cf87748628532a055;
 
-// Account
-const IACCOUNT_ID: felt252 = 0x36c738c1c375b993078fe6b517d477e5a3c9b104e40c04662c4bdd3e2f5fa4a;
+// StandardAccount
+const ISTANDARDACCOUNT_ID: felt252 =
+    0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd;
```
Comment (andrew-fleming) [inline src/openzeppelin/utils/constants.cairo:11]: I know SNIP6 is a draft, but we should include a reference like with `ISRC5` above

```suggestion
// StandardAccount
// See: https://github.com/starknet-io/SNIPs/pull/27
const ISTANDARDACCOUNT_ID: felt252 =
    0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd;
```

**Review by andrew-fleming — COMMENTED** [review]
Looking good, Eric! I left a few questions :)

File: src/openzeppelin/account/interface.cairo [hunk 3]
```diff
@@ -2,8 +2,9 @@ use array::ArrayTrait;
 use array::SpanTrait;
 use starknet::ContractAddress;
 
-const IACCOUNT_ID: felt252 = 0x36c738c1c375b993078fe6b517d477e5a3c9b104e40c04662c4bdd3e2f5fa4a;
-const ERC1271_VALIDATED: u32 = 0x1626ba7e_u32;
+const ISTANDARDACCOUNT_ID: felt252 =
```
[the remark this answers is not in the document]
Comment (ericnordelo) [reply src/openzeppelin/account/interface.cairo:5]: Agree. I will update it, but I tried to be consistent with the IACCESSCONTROL_ID we have. Should we update that as well?

File: src/openzeppelin/account/account.cairo [hunk 4]
```diff
@@ -87,18 +89,22 @@ mod Account {
...
 
-        fn __validate_declare__(class_hash: felt252) -> felt252 {
-            validate_transaction()
-        }
-
-        fn is_valid_signature(message: felt252, signature: Array<felt252>) -> u32 {
+        fn is_valid_signature(message: felt252, signature: Array<felt252>) -> felt252 {
             if _is_valid_signature(message, signature.span()) {
-                ERC1271_VALIDATED
+                VALIDATE_MAGIC_VALUE
```
[the remark this answers is not in the document]
Comment (ericnordelo) [reply src/openzeppelin/account/account.cairo:94]: Ohh, I missed that in corelib. Agree is better to use it. Good call.

File: src/openzeppelin/account/interface.cairo (hunk 1 above)
Comment (ericnordelo) [reply src/openzeppelin/account/interface.cairo:16]: Good point, I think it make sense yes.

File: src/openzeppelin/account/interface.cairo (hunk 3 above)
[the remark this answers is not in the document]
Comment (andrew-fleming) [reply src/openzeppelin/account/interface.cairo:5]: Ahh right! If we're using `ISRC6_ID`, then we get around answering that question here haha but to your point, it's probably better to add the underscore in access control. Setting it as one word is my bad 😅

**Review by andrew-fleming — APPROVED** [review]
I left a tiny suggestion and a comment, but I think this is good to go!

**Pull request merged**